mmf/datasets/builders/localized_narratives/masked_builder.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+from mmf.common.registry import registry
+from mmf.datasets.builders.localized_narratives.masked_dataset import (
+    MaskedLocalizedNarrativesDataset,
+)
+from mmf.datasets.mmf_dataset_builder import MMFDatasetBuilder
+
+
+@registry.register_builder("masked_localized_narratives")
```

Comment:
Are you planning to add normal "localized_narratives" for captioning purposes as well?

Reply:
Yes. They will be in a different PR. :)